e2e-tests/cypress/support/page-objects/invoices.ts

```diff
@@ -0,0 +1,177 @@
+// invoices.ts
```

Comment:
we have SampleData class. Maybe we should add a function that generates invoices in some index there?